Gemfile

```diff
@@ -9,7 +9,7 @@ end
 
 group :test do
   gem "coveralls", require: false
-  gem "rbnacl-libsodium"
+  gem "rbnacl-libsodium", ENV["LIBSODIUM_VERSION"]
```

Comment:
Note that if `ENV["LIBSODIUM_VERSION"]` is unset, this just installs the latest version